x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/shared/map_processor_type_to_form.tsx

```diff
@@ -246,210 +371,373 @@ export const mapProcessorTypeToDescriptor: MapProcessorTypeToDescriptor = {
       defaultMessage: 'Sort',
     }),
-    description: i18n.translate('xpack.ingestPipelines.processors.description.sort', {
+    typeDescription: i18n.translate('xpack.ingestPipelines.processors.description.sort', {
       defaultMessage: "Sorts a field's array elements.",
     }),
+    getDefaultDescription: ({ field, target_field: targetField = field, order = 'asc' }) =>
+      i18n.translate('xpack.ingestPipelines.processors.defaultDescription.sort', {
+        defaultMessage:
+          'Sorts elements in the array "{field}" by order "{order}" to "{target_field}"',
+        values: {
+          field,
+          order,
+          target_field: targetField,
+        },
+      }),
   },
   split: {
     FieldsComponent: Split,
     docLinkPath: '/split-processor.html',
     label: i18n.translate('xpack.ingestPipelines.processors.label.split', {
       defaultMessage: 'Split',
     }),
-    description: i18n.translate('xpack.ingestPipelines.processors.description.split', {
+    typeDescription: i18n.translate('xpack.ingestPipelines.processors.description.split', {
       defaultMessage: 'Splits a field value into an array.',
     }),
+    getDefaultDescription: ({ field, target_field: targetField = field }) =>
+      i18n.translate('xpack.ingestPipelines.processors.defaultDescription.split', {
+        defaultMessage: 'Splits the string stored in "{field}" to an array in "{target_field}"',
+        values: {
+          field,
+          target_field: targetField,
+        },
+      }),
   },
   trim: {
     FieldsComponent: Trim,
     docLinkPath: '/trim-processor.html',
     label: i18n.translate('xpack.ingestPipelines.processors.label.trim', {
       defaultMessage: 'Trim',
     }),
-    description: i18n.translate('xpack.ingestPipelines.processors.description.trim', {
+    typeDescription: i18n.translate('xpack.ingestPipelines.processors.description.trim', {
       defaultMessage: 'Removes leading and trailing whitespace from a string.',
     }),
+    getDefaultDescription: ({ field }) =>
+      i18n.translate('xpack.ingestPipelines.processors.defaultDescription.trim', {
+        defaultMessage: 'Trims whitespaces from "{field}"',
+        values: {
+          field,
+        },
+      }),
   },
   uppercase: {
     FieldsComponent: Uppercase,
     docLinkPath: '/uppercase-processor.html',
     label: i18n.translate('xpack.ingestPipelines.processors.label.uppercase', {
       defaultMessage: 'Uppercase',
     }),
-    description: i18n.translate('xpack.ingestPipelines.processors.description.uppercase', {
+    typeDescription: i18n.translate('xpack.ingestPipelines.processors.description.uppercase', {
       defaultMessage: 'Converts a string to uppercase.',
     }),
+    getDefaultDescription: ({ field }) =>
+      i18n.translate('xpack.ingestPipelines.processors.defaultDescription.uppercase', {
+        defaultMessage: 'Converts value stored in "{field}" to uppercase',
```

Comment:
```suggestion
        defaultMessage: 'Converts values in "{field}" to uppercase',
```

Reply:
This change aligns with the description for [Lowercase](https://github.com/elastic/kibana/pull/95507/files/486008fca8eefac0ba8751f54c33991138771681#diff-89b4d292dc7830b2b872d351c8658ced09588b570c49eb5cb111edf9e7898bbbR507)